workflows/variantcalling.nf

```diff
@@ -63,6 +63,9 @@ workflow VARIANTCALLING {
     INPUT_CHECK (
         ch_input
     )
+
+    INPUT_CHECK.out.reads.view()
+
     ch_versions = ch_versions.mix(INPUT_CHECK.out.versions)
 
     CUSTOM_DUMPSOFTWAREVERSIONS (
```

Comment:
Add comments before this module implementation. Similar to the subworkflow above.

Reply:
done